README.md

```diff
@@ -1,2 +1,44 @@
 # FindYourPet
-Find Your Pet web application.
+Find Your Pet web application Install.
+here is the steps to configurate the project in your pc
+- first you must to install git in your pc.
+# GIT INSTALL 
+* **WINDOWS** :
+1. you will to go at this direccion [press here](https://git-scm.com/download/win") here you be able to install it in your pc.
```

Comment:
In all sentences the first letter is uppercase. 

- To download git please go here*** It's more descriptive